src/wp-includes/fonts/class-wp-font-collection.php

```diff
@@ -134,6 +124,12 @@ private function load_from_file( $file ) {
 			return new WP_Error( 'font_collection_decode_error', __( 'Error decoding the font collection JSON file contents.' ) );
 		}
 
+		$data['name'] = $this->data['name'];
+
+		if ( isset( $this->data['description'] ) ) {
+			$data['description'] = $this->data['description'];
+		}
+
```

Comment:
Does make sense to be able to provide the font collection `name` and `description` from a JSON file?
Probably, we should restrict the JSON files to only a list of font families. Otherwise, you can set a `name`, `categories` and `description` and if the JSON file have those fields the PHP ones will be ignored. Example:

```
wp_register_font_collection(
	'google-fonts',
	array(
		'src'         => 'https://s.w.org/images/fonts/17.7/collections/google-fonts-with-preview.json',
		'name'        => __( 'Custom Fonts (is ignored)' ),
		'description' => __( 'Custom description (is ignored)' ),
	)
);
```

Reply:
I thought I wrote it so that the PHP ones override the ones from the JSON file, not the other way around.

But yeah, good point.

So if we remove those fields from the schema and only keep families, that works.

Downside: the font families in the JSON file refer to categories that are only defined in PHP. Is that an issue?

Reply:
> I thought I wrote it so that the PHP ones override the ones from the JSON file, not the other way around.

Ohh, if the PHP args override the JSON ones the design makes more sense. It's not what's happening right now.

> Downside: the font families in the JSON file refer to categories that are only defined in PHP. Is that an issue?

Technically there are no issues around that. It should work out of the box.

Reply:
I can change it to includ only fsmilies, but then we‘d need to add a new JSON file & schema to the CDN for testing first